dffml/model/model.py

```diff
@@ -84,6 +106,208 @@ def __call__(self) -> ModelContext:
         self._make_config_location()
         return self.CONTEXT(self)
 
+    async def __aenter__(self):
+        if self.config.location.is_file():
```

Comment:
- Let's add two more properties to the model config, Let's check for them in the same way we check of location (`hasattr()`), don't assume that a model config has a location property, don't assume that it has `location_save` or `location_load`. These will be of type DataFlow. Don't go add these to the each model yet. Just implement for SLR model to test first. We'll add to other model configs in next PR.

- Let's take the dataflows you're creating, and put the helper function to create them defined in `dffml/dataflow/archive.py`. When we `run()` the dataflow we can pass the `input_path` and the `output_path`. Let's make sure that we always pass these to the dataflow as some standard definition name. Maybe, `model_location` and `model_tempdir` or something, the flow will need to be updated to grab from these locations for the dataflows your function creates.

Reply:
Done 👍🏽